kibana-reports/public/components/context_menu/context_menu.js

```diff
@@ -264,9 +280,10 @@ const getUuidFromUrl = () =>
   );
 const isDiscover = () => window.location.href.includes('discover');
 
-window.onhashchange = function () {
+$(window).one('hashchange', function( e ) {    
```

Comment:
question, what's the difference between `window.onhashchange = ...` and `$(window).one('hashchange'...`?

Reply:
Before, the `onhashchange` is called twice whenever switching, namely between index patterns in `Discover`. Switching to `$(window).one('hashchange'...` ensures that it is only called once